File: WebApi/api/api/ModelViews/CustomerView.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace api.ModelViews
{
    public class CustomerAutoCompleteSearchView
    {
        public string type { get; set; } // tel, name
        public string txt { get; set; }
    }

    public class CustomerView
    {
        public string customerName { get; set; }
        public string address1 { get; set; }
        public string address2 { get; set; }
        public string district { get; set; }
        public string subDistrict { get; set; }
        public string province { get; set; }
        public string zipCode { get; set; }
        public string tel { get; set; }
    }

}

File: web/src/app/_model/customer-view.ts
import { AppSetting } from "../_constants/app-setting";

export class CustomerView {
    public customerId: number = 0;
    public name: string = "";
    public tel: string = "";
    public addressName: string = "";
    public subDistrict: string = "";
    public district: string = "";
    public province: string = "";
    public zipCode: string = "";
}

export class CustomerAutoCompleteSearchView
{
    
    public type:string = ""; // tel, name
    public txt:string =""; 
}

export class CustomerSearchView
{
    public pageIndex: number = 1;
    public itemPerPage: number = AppSetting.itemPerPage;
    public name:string = "";
}
